### ISSUE TITLE
Is this resource deprecated?

### ISSUE THREAD

**Issue by Happycoil** [PowerShell/PackageManagementProviderResource#36]
Is this resource deprecated in favour of the DSC resources in the PackageManagement module?

https://github.com/OneGet/oneget/tree/master/src/Microsoft.PackageManagement.DscResources

cc @brywang-msft

**Comment by brywang-msft** [comment]
Yes. Any contributions to the DSC resources should/will go to the oneget/oneget repo. All issues prior were also migrated to the repo. Issues created here going forward won't be migrated unless Jianyun or I get poked. PackageManagement 1.1.3.0 onwards contains the DSC resources.

In addition, the PSModule/NuGet specific resources were removed in favor of the generic Package resource. However, I think the team would be open to adding them back in if there's need (and that request/PR should go to the oneget/oneget repo).

**Comment by artisticcheese** [comment]
How do you use DSC to install Modules using Packagemanagement? Built in resource "Package" can only install EXE, MSI and nothing else.

**Comment by jianyunt** [comment]
see example: https://github.com/PowerShell/PackageManagementProviderResource/blob/master/Examples/Sample_PSModule.ps1

**Comment by artisticcheese** [comment]
But this relies on this resource which is depreciated, is not it?

**Comment by brywang-msft** [comment]
https://github.com/OneGet/oneget/blob/WIP/Test/Examples/Sample_Install_Package.ps1

Examples from this repo use the resources available in this repo. Consensus on review was that PSModule and NugetPackage were duplications, so they were removed when merging the DSC resources into the main module.
